mit-open/etl_settings.yml

```diff
@@ -0,0 +1,6 @@
+mit-open:
+  db_url: postgresql://<user>:<password>@<host>:5432/<dbname>
+  s3_bucket: mitodl-data-lake/mailgun
+  hash_salt: 
+  aws_access_key_id: 
+  aws_secret_access_key: 
```

Comment:
Does this file not go into pillar similar to `mitx_etl`?

Reply:
It does, I was just putting this in as an example of the settings that are supported.